atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SerializableTransaction.java

```diff
@@ -253,23 +256,13 @@ public SerializableTransaction(MetricsManager metricsManager,
     }
 
     private ConcurrentNavigableMap<Cell, byte[]> getReadsForTable(TableReference table) {
-        ConcurrentNavigableMap<Cell, byte[]> reads = readsByTable.get(table);
-        if (reads == null) {
-            ConcurrentNavigableMap<Cell, byte[]> newMap = new ConcurrentSkipListMap<>();
-            readsByTable.putIfAbsent(table, newMap);
-            reads = readsByTable.get(table);
-        }
-        return reads;
+        return readsByTable.computeIfAbsent(table, $ -> new ConcurrentSkipListMap<>());
```

Comment:
probably best to not use '$' as a variable name; it's disallowed by our checkstyle but unenforced due to a checkstyle bug. Rather, prefer something like 'unused'.